Curve-finance-suite/Remove-Imbalace-Liquidity/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "forta-agent-starter",
+  "version": "0.0.1",
+  "description": "Forta Agent Typescript starter project",
```

Comment:
`name` needs to be changed to the agent id, you can choose one. We usually use `<protocol>-<agent#>` so `curve-2` will be fine.
Also add a small description of the agent in `description`